### DISCUSSION TITLE
Understanding Stockfish's evaluation

### DISCUSSION THREAD

**Discussion by ghost** [official-stockfish/Stockfish#4754] [Q&A]
👋 Hello there, 

Do the following numbers still stand for the new evaluation function?

| Range | Evaluation | Symbol |
| ----------| --------------- | ----------- |
| -0.26 to 0.26 | The position is equal | = |
| 0.27 to 0.7 | White has a slight advantage | ⩲ |
| 0.7 to 1.5 | White has a moderate advantage | ± |
| > 1.5 | White has a decisive advantage | + − |
| -0.27 to -0.7 | Black has a slight advantage | ⩱ |
| -0.7 to -1.5 | Black has a moderate advantage | ∓ |
| > -1.5 | Black has a decisive advantage | − + |

See https://github.com/chesslablab/php-chess/issues/343

Thanks for the help,

**Comment by cj5716** [comment]
Stockfish's evaluation has been normalized, which means that a 1.0 evaluation stands for 50% chance of win.
I am not sure where you got those values, and whether they are accurate.
https://github.com/official-stockfish/Stockfish/pull/4636 as you can see from here, ~0.5 is a 5% chance of winning, ~0.6 is a 10% chance of winning, ~0.8 is a 25% chance of winning, and so on.
![Screenshot_20230819-072421](https://github.com/official-stockfish/Stockfish/assets/125858804/d41c2dfa-5a19-45eb-948b-d0bd417d4976)

**Comment by vondele** [comment]
Also https://github.com/official-stockfish/Stockfish/wiki/Stockfish-FAQ#interpretation-of-the-stockfish-evaluation

**Comment by ghost** [comment]
[Chess\UciEngine\Stockfish.php](https://github.com/chesslablab/php-chess/blob/master/src/UciEngine/Stockfish.php) is now using the new evaluation function with the ranges described above. See [StockfishTest.php](https://github.com/chesslablab/php-chess/blob/master/tests/unit/UciEngine/StockfishTest.php).

**Comment by DrGrekenstein** [comment]
[official-stockfish/Stockfish#4373](https://github.com/official-stockfish/Stockfish/issues/4373)

 > Before normalization `internal units / 208` is value in pawns. Now instead it divides by NormalizeToPawnValue (initially 348, shortly after 361, and now 394)

So Stockfish's evaluation is unstable and It is not understandable.

**Reply by Sopel97** [reply]
No. It's perfectly understandable, because it has a clearly defined meaning now. It's even explained in the issue you linked.
